projects/zxing-scanner/src/lib/zxing-scanner.component.ts

```diff
@@ -108,7 +108,7 @@ export class ZXingScannerComponent implements OnInit, OnDestroy {
    * True during autostart and false after. It will be null if won't autostart at all.
    */
   @Output()
-  autostarting: EventEmitter<boolean>;
+  autoStarting: EventEmitter<boolean>;
```

Comment:
Sorry, I think that is the Problem with the API. this should be "autostarting" again.